src/osip-integration-test/src/test/java/edu/kit/pse/osip/monitoring/SimulationControllerTest.java

```diff
@@ -0,0 +1,77 @@
+package edu.kit.pse.osip.monitoring;
+
+import edu.kit.pse.osip.core.OSIPConstants;
+import edu.kit.pse.osip.core.io.files.ServerSettingsWrapper;
+import edu.kit.pse.osip.core.io.networking.RemoteMachine;
+import edu.kit.pse.osip.core.model.base.TankSelector;
+import edu.kit.pse.osip.core.opcua.client.UAClientException;
+import edu.kit.pse.osip.monitoring.controller.MixTankClient;
+import edu.kit.pse.osip.monitoring.controller.TankClient;
+import edu.kit.pse.osip.simulation.controller.SimulationController;
+import java.io.File;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.ExecutionException;
+import javafx.stage.Stage;
+import org.junit.AfterClass;
+import org.junit.Test;
+import org.testfx.framework.junit.ApplicationTest;
+
+import static org.junit.Assert.assertTrue;
+
+/**
+ * A class to test the SimulationController class.
+ *
+ * @author David Kahles
+ * @version 1.0
+ */
+public class SimulationControllerTest extends ApplicationTest {
+    private static CompletableFuture<Boolean> closed = new CompletableFuture<>();
+    private static SimulationController simCont;
+
+    @Override
+    public void start(Stage primaryStage) {
+        simCont = new SimulationController();
+        simCont.start(primaryStage);
+    }
+
+    /**
+     * Start the SimulationController and check whether we can connect OPC UA Clients to it. Don't test
+     * the GUI any further. Just make sure it doesn't throw an exception.
+     * @throws InterruptedException If the future has an error.
+     * @throws UAClientException If connecting to the OPC UA servers fails.
+     */
+    @Test(timeout = 30000)
+    public void testLaunch() throws InterruptedException, UAClientException {
+        Thread.sleep(5000);
```

Comment:
Okay man braucht weder sleep noch completable. Fehler in start() werden erkannt. Außerdem wird das von testfx immer zuerst ausgeführt